server/index.js

```diff
@@ -8,6 +8,9 @@ import habitRouter from "./src/routes/habitRoutes.js";
 import answerRouter from "./src/routes/answerRoutes.js";
 import verifyToken from "./src/middleware/auth.js";
 
+// Handle Unexpected error module
+import { unexpectedErrorHandler } from "./src/middleware/catch.js";
```

Comment:
- should we rename the function as just `errorHandler`, I think that would be simpler.
- also can we make the filename same as the function name? instead of `catch.js`, let's make it `errorHandler.js`

Reply:
Yes, changed